types/node/fs.d.ts

```diff
@@ -2791,7 +2791,8 @@ declare module 'fs' {
         persistent?: boolean | undefined;
         recursive?: boolean | undefined;
     }
-    export type WatchListener<T> = (event: 'rename' | 'change', filename: T) => void;
+    export type EventType = 'rename' | 'change';
```

Comment:
Actually can you rename this to `WatchEventType`, `EventType` is very generic.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55033/commits/7410c0f73d76a6721532ccb4b956fe9b3e7c424a